e2e/dragonfly_pod_lifecycle_controller_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"time"
 
 	dfv1alpha1 "github.com/dragonflydb/dragonfly-operator/api/v1alpha1"
+	dragonflydbiov1alpha1 "github.com/dragonflydb/dragonfly-operator/api/v1alpha1"
```

Comment:
Why do we need the same import with different import name (`dfv1alpha1` and `dragonflydbiov1alpha1`?